src/connectors/tagging/taginput/taginput.js

```diff
@@ -150,11 +159,9 @@ Taginput.propTypes = {
   setFocus: PropTypes.func,
   addTag: PropTypes.func,
   handleRemoveAction: PropTypes.func,
-  handleDirection: PropTypes.func,
   makeTagsInactive: PropTypes.func,
   setValue: PropTypes.func,
-  inputRef: PropTypes.func,
+  // inputRef: PropTypes.func,
```

Comment:
We can delete this now that it's being passed in from `Tagging` right?

Reply:
This was due to the fact that a react created `ref` isn't a function. We should probably update it once this file gets overhauled with linaria